### PR TITLE
feat(17/WAKU-RLN-RELAY): add RLN credentials format

### PR DESCRIPTION
**Pull request by jimstir**
This PR proposes a JSON format for RLN credentials that can be easily generalized to different applications. It supports multi-groups for same (id_key, id_commitment) pair and multiple id pairs per application.

This PR addresses https://github.com/vacp2p/rfc/issues/543

Reference pull request: https://github.com/vacp2p/rfc/pull/550

### CODE REVIEW

File: waku/standards/core/17/rln-relay.md [hunk 1]
```diff
@@ -6,11 +6,12 @@ status: draft
 tags: waku-core
 editor: Sanaz Taheri <sanaz@status.im>
 contributors:
-  - Oskar Thorén <oskarth@titanproxy.com>
```
Comment (rymnc) [inline waku/standards/core/17/rln-relay.md:9]: I think Oskar wishes this email is used, not the status.im one

File: waku/standards/core/17/rln-relay.md [hunk 2]
```diff
@@ -193,6 +183,43 @@ Below is the description of the fields of `RateLimitProof` and their types.
 | `share_x` and `share_y`| array of 32 bytes each | Shamir secret shares of the user's secret identity key `sk` . `share_x` is the Poseidon hash of the `WakuMessage`'s `payload` concatenated with its `contentTopic` . `share_y` is calculated using [Shamir secret sharing scheme](/spec/32) | <!-- todo specify the poseidon hash setting -->
 | `nullifier`  | array of 32 bytes | internal nullifier derived from `epoch` and peer's `sk` as explained in [RLN construct](/spec/32)|
 
+## RLN Credentials format
```
Comment (rymnc) [inline waku/standards/core/17/rln-relay.md:186]: shouldn't this be in the waku keystore RFC?

File: waku/standards/core/17/rln-relay.md (hunk 1 above)
Comment (jimstir) [reply waku/standards/core/17/rln-relay.md:9]: @rymnc some of the pr being transferred from the old repo will not be updated. I have not looked over any of these, before merging everything should be checked.

**Comment by jimstir** [comment]
Discussion continued on https://github.com/vacp2p/rfc-index/pull/32

**Pull request closed without merging**